docs/tasks/tools/install-kubectl.md

```diff
@@ -129,6 +129,14 @@ echo "source <(kubectl completion bash)" >> ~/.bash_profile
 Please note that this only appears to work currently if you install using `brew install kubectl`,
 and not if you downloaded kubectl directly.
 
+### On MacOS, using ZSH
+
+To add kubectl autocomplete to your ZSH profile:
+
+```shell
+sed "s/^\(plugins=(\)\(.*\)\()\)$/\1\2 kubectl\3/" ~/.zshrc
```

Comment:
I'm -1 for this. It's a very opaque command most users don't understand. First of all, what happens if there's no `plugins=` in .zshrc.

I also have a question: Have you tested this without oh-my-zsh? I believe plugins are an oh-my-zsh concept and not `zsh` so I've reason to believe this command assumes oh-my-zsh.